src/azure-cli/azure/cli/command_modules/servicefabric/_help.py

```diff
@@ -581,6 +581,15 @@
         az sf managed-node-type vm-extension add -g testRG -c testCluster -n snt --extension-name csetest --publisher Microsoft.Compute --extension-type BGInfo --type-handler-version 2.1 --auto-upgrade-minor-version
 """
 
+helps['sf managed-node-type vm-extension update'] = """
+type: command
+short-summary: update an existing extension on the node type.
+examples:
+  - name: update bg extension.
+    text: >
+        az sf managed-node-type vm-extension update -g testrg -c testcluster -n pnt --extension-name testextension --type-handler-version 2.1 --auto-upgrade-minor-version --force-update-tag forced-update
```

Comment:
can you add another example of updating extension settings? that is a more common scenario for customers.

Reply:
and also one for using the new params: --provision-after-extension and --setup-order

Reply:
updated